### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by rodg1k**
[DEMO LINK](https://rodg1k.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://rodg1k.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,74 @@
...
+  padding: 20px 10px;
+  border-bottom: 4px solid transparent;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  background-color: #00acdc;
+  height: 4px;
+  width: 0%;
```
[the remark this answers is not in the document]
Comment (DarkMistyRoom) [reply src/style.css:61]: this line should always be visible

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,68 @@
...
+        <a href="index.html">
+          <img src="logo.png" alt="logo">
+        </a>
+      </div>
+
+      <div class="nav">
+        <ul class="nav__list">
+
+          <li>
+            <a class="nav__link is-active" href="#">Apple</a>
```
Comment (DarkMistyRoom) [inline src/index.html:34]:
```suggestion
            <a class="nav__link is-active" href="#apple">Apple</a>
```

File: src/index.html [hunk 3]
```diff
@@ -3,12 +3,68 @@
...
+          <li>
+            <a class="nav__link is-active" href="#">Samsung</a>
+          </li>
+
+          <li>
+            <a class="nav__link is-active" href="#">Smartphones</a>
+          </li>
+
+          <li>
+            <a class="nav__link is-active" data-qa="hover" href="#">Laptops & Computers</a>
```
[the remark this answers is not in the document]
Comment (DarkMistyRoom) [reply src/index.html:46]: image

File: src/index.html [hunk 4]
```diff
@@ -3,12 +3,98 @@
...
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    >
```
Comment (anstsot) [inline src/index.html:13]:
```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com">
```

File: src/index.html [hunk 5]
```diff
@@ -3,12 +3,98 @@
...
+      rel="stylesheet"
+    >
   </head>
+  
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo"
+        href="index.html"
+      >
```
Comment (anstsot) [inline src/index.html:24]: please fox the code styles
```suggestion
      <a class="logo" href="index.html">
```

**Review by anstsot — APPROVED** [review]
Great job, approved!

**Pull request closed without merging**